portals/knowledge-management/src/Pages/ViewArticle.vue

```diff
@@ -85,9 +94,23 @@
                 article.value = response.data.article;
                 loading.value = false;
                 portalViewCount.value = response.data.portal_view_count;
+                feedback.value = response.data.article.vote ? response.data.article.vote.is_helpful : null;
+                articalWasHelpfulFeatureFlag.value = response.data.article_was_helpful_feature_flag;
             },
         );
     }
+    async function toggleFeedback(type) {
+        if (feedback.value === type) {
+            feedback.value = null; // Unselect if the same button is clicked
+        } else {
+            feedback.value = type; // Select the clicked button
+        }
```

Comment:
This should probably be actually updated in a `then` of the post request.

Because what would happen right now if the `post` request failed? The user would thing their vote went through as it looks that way on the frontend. But nothing actually happened on the backend.